src/lib/DepositRecordSerializer.js

```diff
@@ -97,6 +98,19 @@ export class DepositRecordSerializer {
     }),
   };
 
+  /**
+   * Return Field matching given fieldpath
+   * @method
+   * @param {string} fieldpath - dotted path to field in metadata
+   * @returns {Field} corresponding field
+   */
+  getField(fieldpath) {
```

Comment:
Isn't it the same as `lodash/get` function?

Reply:
yes! Must have been the end of the week.

Reply:
Forgot to move to `lodash/get` :)